app/src/main/java/org/fossasia/openevent/app/common/app/binding/BindingAdapters.java

```diff
@@ -165,7 +163,7 @@ public static void setTintColor(ImageView imageView, @ColorInt int color) {
 
     @BindingAdapter("backgroundTint")
     public static void setBackgroundTintColor(View view, @ColorInt int color) {
-        ViewCompat.setBackgroundTintList(view, ColorStateList.valueOf(color));
+        view.setBackgroundColor(color);
```

Comment:
This is not the correct way to set tint. And this doesn't even need a layout binding. You can directly use `backgroundColor` in XML